Resources/Prototypes/Entities/Clothing/Back/satchel.yml

```diff
@@ -94,6 +94,50 @@
   - type: Clothing
     sprite: Clothing/Back/Satchels/hydroponics.rsi
 
+- type: entity
+  parent: ClothingBackpackSatchel
+  id: ClothingBackpackSatchelVirology
+  name: virologists satchel
+  description: A sterile satchel with virologist colours.
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Satchels/virology.rsi
+  - type: Clothing
+    sprite: Clothing/Back/Satchels/virology.rsi
+
+- type: entity
+  parent: ClothingBackpackSatchel
+  id: ClothingBackpackSatchelGenetics
+  name: hydroponics satchel
```

Comment:
fix

Reply:
From my nuked repository so unable to edit now unless the PR can be pointed to a new location? My bad but shouldn't happen again.